### ISSUE TITLE
Dm save issue

### ISSUE THREAD

**Issue by aleondopeso** [JMccormick264/InstagramPP#3]
When i try to download a temporal photo or a temporal video wich i recieve by a direct, the download button didn’t work, the keyboard open automatically to repply

**Comment by afonso177** [comment]
I'm experiencing a similar issue, the downloaad button didn't even appear, and the autosave function for that didn't work

**Comment by rikianastasoski** [comment]
same here @JMccormick264

**Comment by JMccormick264** [comment]
Hmm. I have that on my smaller device. What device are you all using?

**Comment by aleondopeso** [comment]
@JMccormick264 i’m using an iPhone 7 Plus

**Comment by afonso177** [comment]
@JMccormick264 I'm using an iPhone 6S

**Comment by JMccormick264** [comment]
And for all of you the download and mark as read is in the text box? So when you click on one of those buttons it opens the reply box instead of the buttons correct?

**Comment by aleondopeso** [comment]
Yes, exactly @JMccormick264

**Comment by rikianastasoski** [comment]
No, i dont have any buttons and disable auto advance and ss alert dont work

**Comment by JMccormick264** [comment]
hmm @rikianastasoski I Will make a separate issue report for that. Sideload correct?

**Comment by aleondopeso** [comment]
And for me? Any solution? @JMccormick264

**Comment by rikianastasoski** [comment]
yes

**Comment by afonso177** [comment]
Yes sideload

**Comment by aleondopeso** [comment]
I’ve sideloaded the last version and the bug continues

**Comment by aleondopeso** [comment]
@JMccormick264 any fix?

**Comment by afonso177** [comment]
Still no fix?

**Comment by JMccormick264** [comment]
Can you try the most recent release?

**Comment by afonso177** [comment]
Still happening with the latest release

**Comment by afonso177** [comment]
@JMccormick264 I've seen this bug has been fixed in the latest version of the tweak apperantly (the one available for Cydia), so could you update the one available as an .ipa for Instagram ++? (And Snapchat ++ because the current latest .ipa doesn't let me sign in)
